uefi-raw/CHANGELOG.md

```diff
@@ -5,7 +5,14 @@
 - Added `PciRootBridgeIoProtocol`.
 - Added `ConfigKeywordHandlerProtocol`.
 - Added `HiiConfigAccessProtocol`.
+- Added lots of convenient methods and `From` implementations to better
+  integrate the `uefi_raw::net::*` types with the types from `core::net::*`.
+  Further, more associated functions for an improved convenience have been
+  added to the types of that module.
 
+## Changed
+- Types `MacAddress`, `IpAddress`, `Ipv4Address`, and `Ipv6Address` were moved
+  from `uefi_raw::*` to `uefi_raw::net::*`.
```

Comment:
I think moving the implementation to a new module makes sense, but I think for the public API it might make sense to let that be an internal detail, i.e. keep the `net` module private and pub-export the types from the crate root. These types are listed in the [Common UEFI Data Types](https://uefi.org/specs/UEFI/2.11/02_Overview.html#data-types) portion of the spec, which to mind means they fit best at the top level (and this would also avoid making the change API breaking).

Reply:
I agree, makes sense!